### DISCUSSION TITLE
Missing "Edit"-icon after recent update to version 9

### DISCUSSION THREAD

**Discussion by AndreaEsposit** [squidfunk/mkdocs-material#4871] [Q&A] [answered]
Hey, 
I have recently updated a documentation page I have to version 9. I noticed however that the edit-icon is missing. Is there a way to bring it back? 

To be more precise, I'm referring to the icon that you used to open an MD file in edit mode on Github.

**Comment by squidfunk** [comment, marked as answer]
Did you consult our [upgrade guide](https://squidfunk.github.io/mkdocs-material/upgrade/) when upgrading from 8 to 9? Specifically the [section on the edit button](https://squidfunk.github.io/mkdocs-material/upgrade/#contentaction).

**Reply by AndreaEsposit** [reply]
Thank you :), will take a look at it. I see that in version 9 it needs to be explicitly enabled. I took a look at the release notes and at the CHANGELOG file. I wasn't aware of that guide.

**Reply by squidfunk** [reply]
I'll add a link to the upgrade guide in the changelog the next time we release a major version then, makes sense!
